.circleci/config.yml

```diff
@@ -20,4 +22,19 @@ jobs:
               echo "Awesome! GCS integration tests are enabled."
             fi
       - run: make format
-      - run: make test
\ No newline at end of file
+      - run: make test
+      - run: make docker
+      - run:
+          name: Run tests on generated Docker image
+          command: |
+            docker run improbable/thanos --help
+            docker run improbable/thanos sidecar --help
```

Comment:
Cool! so can we kill this subcommands? I don't thing they test anything.

Reply:
sure, will commit now.